api/stream-relay/index.js

```diff
@@ -503,7 +503,7 @@ export const wrap = dgram => {
     getState () {
       return {
         config: this.config,
-        data: [...this.cache.data.values()],
+        data: [...this.cache.data.entries()],
```

Comment:
why `[...this.cache.data.entries()]` and not just `this.cache.data.entries()`?

Reply:
`map.entries()` returns `[Map Entries]` where as `[...map.entries()]` returns an array which means one less type of thing to deal with at serialization time.